index.js

```diff
@@ -80,6 +84,19 @@ module.exports = function (opts, minimistOpts) {
 		flags: camelcaseKeys(argv),
 		pkg: pkg,
 		help: help,
-		showHelp: showHelp
+		showHelp: showHelp,
+		stdinOrInput: function () {
+			if (!_ && process.stdin.isTTY) {
+				return Promise.reject(new Error('input required'));
+			}
+
+			if (_.length > 0) {
+				return Promise.resolve(_);
+			}
+
+			return getStdin().then(function (data) {
+				return [data];
```

Comment:
Because `_` is an array as well. This way you can just iterate over the result instead of checking `Array.isArray`.